src/Microsoft.Data.SqlClient/tests/ManualTests/DataCommon/ConnectionTestParametersData.cs

```diff
@@ -0,0 +1,336 @@
+            yield return new object[] { Data.ConnectionTestParametersList[15] };
+            yield return new object[] { Data.ConnectionTestParametersList[16] };
+            yield return new object[] { Data.ConnectionTestParametersList[17] };
+            yield return new object[] { Data.ConnectionTestParametersList[18] };
+            yield return new object[] { Data.ConnectionTestParametersList[19] };
+            yield return new object[] { Data.ConnectionTestParametersList[20] };
+            yield return new object[] { Data.ConnectionTestParametersList[21] };
+            yield return new object[] { Data.ConnectionTestParametersList[22] };
+            yield return new object[] { Data.ConnectionTestParametersList[23] };
+            yield return new object[] { Data.ConnectionTestParametersList[24] };
+            yield return new object[] { Data.ConnectionTestParametersList[25] };
+            yield return new object[] { Data.ConnectionTestParametersList[26] };
+            yield return new object[] { Data.ConnectionTestParametersList[27] };
+            yield return new object[] { Data.ConnectionTestParametersList[28] };
+            yield return new object[] { Data.ConnectionTestParametersList[29] };
+        }
+
+        public ConnectionTestParametersData() 
+        {
+            ConnectionTestParametersList = new List<ConnectionTestParameters>
+            {
+                new ConnectionTestParameters
+                {
+                    TdsEncryptionType = TDSPreLoginTokenEncryptionType.Off,
+                    Encrypt = SqlConnectionEncryptOption.Optional,
+                    TrustServerCertificate = false,
+                    Certificate = null,
+                    HostNameInCertificate = null,
+                    TestResult = true
+                },
+                new ConnectionTestParameters
+                {
+                    TdsEncryptionType = TDSPreLoginTokenEncryptionType.Off,
+                    Encrypt = SqlConnectionEncryptOption.Mandatory,
+                    TrustServerCertificate = false,
+                    Certificate = null,
+                    HostNameInCertificate = null,
+                    TestResult = false
+                },
+                new ConnectionTestParameters
+                {
+                    TdsEncryptionType = TDSPreLoginTokenEncryptionType.Off,
+                    Encrypt = SqlConnectionEncryptOption.Optional,
+                    TrustServerCertificate = true,
+                    Certificate = null,
+                    HostNameInCertificate = null,
+                    TestResult = true
+                },
+                new ConnectionTestParameters
+                {
+                    TdsEncryptionType = TDSPreLoginTokenEncryptionType.Off,
+                    Encrypt = SqlConnectionEncryptOption.Mandatory,
+                    TrustServerCertificate = true,
+                    Certificate = null,
+                    HostNameInCertificate = null,
```

Comment:
same here

Reply:
Sorry could you clarify what same here means?

Reply:
https://github.com/dotnet/SqlClient/pull/2439#discussion_r1571201554

Reply:
"same here" I guess has been addressed.